cmd/cbuild/commands/list/list_configurations.go

```diff
@@ -20,23 +19,22 @@ var ListConfigurationsCmd = &cobra.Command{
 	Short: "Print list of configurations in a csolution.yml",
 	Args:  cobra.ExactArgs(1),
 	RunE: func(cmd *cobra.Command, args []string) error {
-		fileExtension := filepath.Ext(args[0])
-		if !(fileExtension == ".yml" || fileExtension == ".yaml") {
-			return errors.New("invalid file argument")
+		if _, err := os.Stat(args[0]); os.IsNotExist(err) {
```

Comment:
It is not clear to me why we are no longer checking the extensions of the file being specified?

Reply:
This is removed because this is checked by `csolution` as well . But seems `csolution` doesn't seems to check this in all the input scenarios. I am looking into this.